Documentation/sources/propositions/developer-documentation/classes/content-card.md

```diff
@@ -0,0 +1,55 @@
+# ContentCard
+
+An object representing the default content card created in the Adobe Journey Optimizer UI. 
+
+Content cards must be rendered by the app developer.  Tracking a content card is done via calls to the [`track(_:withEdgeEventType:)`](#track_withedgeeventtype) API.
+
+```swift
+@objc(AEPContentCard)
+@objcMembers
+public class ContentCard: NSObject, Codable {
+    /// Plain-text title for the content card
+    public let title: String
+
+    /// Plain-text body representing the content for the content card
+    public let body: String
+
+    /// String representing a URI that contains an image to be used for this content card
+    public let imageUrl: String?
+
+    /// Contains a URL to be opened if the user interacts with the content card
+    public let actionUrl: String?
+
+    /// Required if `actionUrl` is provided. Text to be used in title of button or link in content card
+    public let actionTitle: String?
+
+    ...
+}
+```
+
+# Public functions
+
+## track(_:withEdgeEventType:)
+
+Tracks an interaction with the given `ContentCard`.
+
+```swift
+public func track(_ interaction: String? = nil, withEdgeEventType eventType: MessagingEdgeEventType)
```

Comment:
Is it too late to reverse the order of parameters in the API? `track(withEdgeEventType eventType: MessagingEdgeEventType, _ interaction: String? = nil)` so its easier to not pass the optional value if not needed?

Reply:
nevermind, `PropositionItem`'s track looks the same, we can't change that now